unison-src/transcripts/api-summaries.output.md

```diff
@@ -23,9 +23,28 @@ structural ability Stream s where
 
 ```api
 --  term
-GET /api/definitions/terms/qualified/nat@qkhkl0n238/summary
+GET /api/definitions/terms/by-hash/@qkhkl0n238/summary?name=nat
 {
-    "fqn": "nat",
+    "displayName": "nat",
```

Comment:
We call the name field on `getDefinition` "bestName" currently. I'd love to clean up that API to also use `displayName`, but that shouldn't be too bad.